src/status_im/ui/components/status_bar/styles.cljs

```diff
@@ -0,0 +1,101 @@
+(ns status-im.ui.components.status-bar.styles
+  (:require [status-im.ui.components.styles :as styles]
+            [status-im.utils.platform :as platform])
+  (:require-macros [status-im.utils.styles :refer [defstyle]]))
+
+;; :main
+(defstyle status-bar-main
+  {:ios     {:background-color styles/color-white
+             :bar-style        "default"}
+   :android {:background-color "transparent"
+             :translucent      true
+             :bar-style        "dark-content"}})
+
+(def view-main
+  {:background-color styles/color-white
+   :height           (get platform/platform-specific :status-bar-default-height)})
+
+;; :transparent
+(defstyle status-bar-transparent
+  {:bar-style "light-content"
+   :ios       {:background-color styles/color-transparent}
+   :android   {:background-color "transparent"
+               :translucent      true}})
+
+(def view-transparent
+  {:background-color styles/color-transparent
+   :height           (get platform/platform-specific :status-bar-default-height)})
```

Comment:
Can `status-bar-default-height` be defined in this file somehow?

Reply:
You mean using `defstyle`? But the value is being used in some other places not as the `height` property. Should I need to create a new `defstyle` for each different property with that value?

Reply:
👍